addon/templates/components/drag-sort-occluded.hbs

```diff
@@ -0,0 +1,36 @@
+{{#vertical-collection
+  items
```

Comment:
We are implementing '`vertical-collection` addon. Therefore, items will be the first positional parameter, it can be null / array. 
https://github.com/html-next/vertical-collection/blob/master/addon/components/vertical-collection/component.js#L52